browsing.md

```diff
@@ -38,92 +57,171 @@ and now want to perform the same search for your Urdu deck. You can
-to reverse the sort order. Not all columns can be sorted on.
-
-The due column behaves differently for different types of cards. New
-cards show a number rather than a due date, which indicates the order
-the new cards will be presented in. Cards in (re)learning and reviews
-will both show a due date, but when sorting they are first grouped by
-type and then sorted by date.
-
-The "edited" and "changed" columns sound the same but track different
-things. "Edited" tracks the last time changes were made to the _note_
-(e.g., when the content of a field was edited), while "changed" tracks
-the last time changes were made to the _card_ (e.g., when you reviewed
-the card and the review history and interval were updated).
-
-When you click on a card, its note will be shown in the bottom section.
-If you drag the mouse or hold <kbd>Ctrl</kbd> or <kbd>Command</kbd> to select multiple cards,
-the editor will be temporarily hidden. Various operations (such as
-changing the deck) can operate on multiple cards at once.
-
-The background color will change depending on the card. Marked cards
-are a shade of purple. Suspended cards are a shade of yellow. For more
-information about marked and suspended cards, see [Editing and
-More](studying.md#editing-and-more).
+### Rows
 
-One of the available columns is called the **Sort Field**. Anki allows you
-to choose one field from each type of note to be used for sorting. You
-can change the sort field by clicking **Fields** in the current note
-section.
+If you drag the mouse or hold <kbd>Ctrl</kbd> or <kbd>Command</kbd> to select multiple
+rows, the editor will be temporarily hidden. Various operations (such as
+changing the deck) can operate on multiple cards or notes at once, independent
+of the active mode. Therefore in Cards mode, a note is considered to be selected
+if any of its cards is selected, and in Notes mode, a card is considered to be selected
+if its note is selected.
 
-The question and answer columns display what you’d see on the question
-and answer while reviewing, except the answer column will strip the
-question part for clarity. You can also choose a [custom
-format](templates/styling.md#browser-appearance) in the card type editor instead of showing
-what would be seen during review.
+Other operations (like showing card info) only operate on a single, the so-called
+*current*, card or note, which is usually the one that was last left- or rightclicked.
+In Cards Mode, again, the current note is the note of the current card and in Notes
+mode, the current card is the first card of the current note.
 
-## Current Note
+The background color will change depending on the card and note.
+In Cards mode, Anki will change the color of the row depending on the state of
+the card. The first match will be used: 1. if flagged, use the flag color; 2. if
+suspended, yellow; 3. if marked, purple. In Notes mode, color is only applied to
+marked notes.  
```

Comment:
Maybe this would be a little easier to read as a separate list, instead of contained in the paragraph?